frontend/src/components/elements/ExceptionElement/ExceptionElement.tsx

```diff
@@ -31,29 +32,55 @@ class ExceptionElement extends React.PureComponent<Props> {
   public render(): React.ReactNode {
     const { element, width } = this.props
     const type = element.get("type")
-    let message = element.get("message")
-    if (message) {
-      message = `: ${message}`
-    }
+    const message = element.get("message")
     const stackTrace = element.get("stackTrace")
 
-    // Put it all together into a nice little html view.
+    // Build the message display.
+    // On the backend, we use the StreamlitException type for errors that
+    // originate from inside Streamlit. These errors have Markdown-formatted
+    // messages, and so we wrap those messages inside our Markdown renderer.
+    let messageNode: ReactNode
+    if (element.get("messageIsMarkdown")) {
+      let markdown = `**${type}**`
+      if (message) {
+        markdown += `: ${message}`
+      }
+      messageNode = <StreamlitMarkdown source={markdown} allowHTML={false} />
+    } else {
+      messageNode = (
+        <>
```

Comment:
@tconkling, can you use Fragment instead?

Reply:
I might be misunderstanding what you're asking, but `<>` is a shorthand for `<React.Fragment>`: https://reactjs.org/blog/2017/11/28/react-v16.2.0-fragment-support.html.

Reply:
Wait, I think @kantuni told me a few times to move from <`Fragment`> to `<>`? Or was it someone else? :smile: 

Personally, I have no horse in this race. And I don't even care about keeping a unified style in our codebase for this, except for: it should be the same style within single file.

Reply:
Since this seems to be the preferred shorthand for `Fragment`, I'm going to stick with `<>` until someone tells me otherwise :)